### PR TITLE
[6885] Info section 1

### PR DESCRIPTION
**Pull request by chris-lawton**
For https://github.com/mozilla/foundation.mozilla.org/issues/6885

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/standout-stat.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+
+<div class="standout-stat {% if classes %}{{ classes }}{% endif %}">
+  {% if heading %}
+    <p class="standout-stat__heading mb-0">{{ heading }}</p>
+  {% endif %}
+
+  <div class="standout-stat__wrap text-center p-4">
+    <p class="standout-stat__number mb-2">{{ number }}</p>
```
Comment (helenb) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/standout-stat.html:9]: The numbers aren't using countup yet -  but I can add that as part of my work on section 2.

File: source/js/foundation/pages/youtube-regrets/recommendations-pie-chart.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { Chart, registerables } from 'chart.js';
+
+export const initYouTubeRegretsRecommendationsPieChart = () => {
+  Chart.register(...registerables);
+
+  const categoriesChart = document.getElementById('recommendations-pie-chart');
+  const ctx = categoriesChart.getContext('2d');
+
+  const chart = new Chart(ctx, {
+    type: 'doughnut',
```
Comment (helenb) [inline source/js/foundation/pages/youtube-regrets/recommendations-pie-chart.js:10]: I don't see any labels at all on the chart. I know we said that we would not be able to match the label style exactly but I think we will want some labels showing.

**Review by helenb — COMMENTED** [review]
Thanks @chris-lawton looks good! I'd like to get this merged asap as I need to share some code for info section 2.

**Comment by chris-lawton** [comment]
Thanks for the review @helenb! Let's pick up any changes from the comments in the integration branch so this doesn't block you

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+
+  <p class="mb-5 pb-3">{% trans "YouTube Regrets are primarily a result of the recommendation algorithm, meaning videos that YouTube chooses to amplify, rather than videos that people sought out." %}</p>
+  <h2 class="heading-2 mb-5">{% trans "71% of all regret reports came from videos recommended to our users and recommended videos were 1.4 times more likely to be regretted than videos searched for. " %}</h2>
```
Comment (helenb) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html:4]: Just also noticed: this needs to be an h3 (or maybe not a heading at all) as using an h2 breaks heading structure at the moment.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
```
[the remark this answers is not in the document]
Comment (helenb) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html:1]: resolved in https://github.com/helenb/foundation.mozilla.org/tree/feature/6886-information-section-2

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html (hunk 3 above)
Comment (helenb) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html:4]: resolved in https://github.com/helenb/foundation.mozilla.org/tree/feature/6886-information-section-2

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+  <p class="mb-5 pb-3">{% trans "YouTube Regrets are primarily a result of the recommendation algorithm, meaning videos that YouTube chooses to amplify, rather than videos that people sought out." %}</p>
+  <h2 class="heading-2 mb-5">{% trans "71% of all regret reports came from videos recommended to our users and recommended videos were 1.4 times more likely to be regretted than videos searched for. " %}</h2>
+  <div class="col-12 col-lg-8 mx-auto">
+    <canvas class="" width="410" height="410" id="recommendations-pie-chart"></canvas>
+  </div>
+  {% blocktrans %}
+    <p class="mt-5 mb-lg-5 pb-lg-5">The regrets our volunteers flagged had tons of views, but it’s impossible to tell how many of those views came from YouTube’s recommendation algorithm, because <span class="fw-bold">YouTube won’t release this information</span>.</p>
```
[the remark this answers is not in the document]
Comment (helenb) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion-section-1.html:9]: resolved in https://github.com/helenb/foundation.mozilla.org/tree/feature/6886-information-section-2

**Pull request merged**